.gitignore

```diff
@@ -1,3 +1,7 @@
 .DS_Store
 **node_modules/**
-.vscode
\ No newline at end of file
+.vscode
+dist
+package-lock.json
+permission-portal-cloud-functions
```

Comment:
I think I have some files in my repo from a way earlier version of the repo that have since been removed from `dev`.  And my git pulls don't update them so just making sure that they don't get committed by accident.  

I was thinking maybe I should just clone the `dev` repo again and start over instead of using this .gitignore stuff.